### PR TITLE
Update the header color for badger.scss

### PR DESCRIPTION
**Pull request by mitche50**

### CODE REVIEW

**Review by gokaiorg — APPROVED** [review]

**Comment by gokaiorg** [comment]
There is a conflict in your PR, can you resolve it?

File: skins/badger.scss [hunk 1]
```diff
@@ -5,5 +5,10 @@
...
   --heading-color: #FFFFFF;
   --border-color: #F2A52B;
-  --header-bg: #28283F;
+  --header-bg: #1f1f29;
+}
+
+.button--active {
+  color: #fff !important;
+  background-color: var(--primary-color) !important;
 }
```
Comment (gokaiorg) [inline skins/badger.scss:14]: Please use only theme variables. No extra custom styles are allowed except on #topnav. Thank you.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by mitche50** [comment]
Is there any way to make the active button show a different color? Right now it’s impossible to see which button is selected.

**Comment by gokaiorg** [comment]
I think your best way is to change the border-color, make sure it's not the same as the link-color. Tell me if you can do something in this way, thank you.

**Comment by mitche50** [comment]
Updated to modify link color and remove the extra custom style @gokaiorg

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**